components/eam/src/dynamics/se/inidat.F90

```diff
@@ -51,6 +51,13 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
                                        cldera_sai_tracers_init_cnst
     use cldera_passive_tracers,  only: cldera_passive_tracers_implements_cnst, &
                                        cldera_passive_tracers_init_cnst
+    use cldera_dynamic_tracers,  only: cldera_dynamic_tracers_implements_cnst, &
+                                       cldera_dynamic_tracers_init_cnst, &
+                                       cldera_dynamic_tracers_is_pt, &
+                                       cldera_dynamic_tracers_is_pv, &
+                                       cldera_dynamic_tracers_is_enabled
+    use element_ops,             only: get_pot_vort, &
```

Comment:
These do not appear to be used in this routine, please remove from use list.

Reply:
Most of them do seem to be used; I removed only `cldera_dynamic_tracers_is_enabled`, `get_pot_vort`, and `get_field`